### PR TITLE
Fixes login with Tequila

### PR DESCRIPTION
**Pull request by nkcr**
In short the proxy wasn't working because the extension to `setupProxy` must be `.js`.

- adds 404 message
- applies automatic formatting on some files
- fixes the login callback with tequila
- enables HTTPS by default
- fixes typo "personnal_info"
- adds a conditional import of mocks
- fixes `setupProxy.js` that can't be anything else than a `.js`
- sets strict to false, we are not there yet
- updates the README.md

### CODE REVIEW

File: web/frontend/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "HTTPS=false react-scripts start",
+    "start": "HTTPS=true react-scripts start",
```
Comment (jbsv) [inline web/frontend/package.json:6]: does it still work with the mocked backend ?

File: web/frontend/src/pages/Login.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { FC, useState } from 'react';
+import React, { FC, useEffect, useState } from 'react';
```
Comment (jbsv) [inline web/frontend/src/pages/Login.tsx:1]: useEffect not needed

**Review by jbsv — APPROVED** [review]
well done for the proxy (and the rest). 
minor things maybe worth looking at before merging

File: web/frontend/package.json (hunk 1 above)
Comment (nkcr) [reply web/frontend/package.json:6]: yes, I tried both and it works

**Pull request merged**